### DISCUSSION TITLE
Welcome to erase-install Discussions!

### DISCUSSION THREAD

**Discussion by grahampugh** [grahampugh/erase-install#48] [General]
## 👋 Welcome!
  We’re trying Discussions as a place to give help regarding this tool. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded 🎗️ 

You can also go ahead and ask questions in the [MacAdmins.org Slack Workspace](https://www.macadmins.org), in the `#eraseinstall` channel. Please note that this channel is shared with people interested in the [EraseInstall application](https://bitbucket.org/prowarehouse-nl/erase-install/src/master/), so don't forget to specify which tool you are asking about 😙

**Comment by mani2care** [comment]
https://stackoverflow.com/questions/40108569/how-to-get-the-last-line-of-a-file-using-cat-command

can use the option to show the download status
